### ISSUE TITLE
add more test data (company names)

### ISSUE THREAD

**Issue by petri** [psolin/cleanco#17]
@psolin , would you have any lists of company names that you want to see tested?

**Comment by davidheryanto** [comment]
Hi I've some compay name such as:
- xxx co ltd
- xxx private limited
- xxx pte limited
- xxx co limited

Do you think it's a good idea to add these additional terms on termdata.py ?

**Comment by petri** [comment]
https://opencorporates.com could be used for testing?

**Comment by petri** [comment]
@davidheryanto it depends. What countries are those for?

**Comment by petri** [comment]
I have added a companies.csv file to the tests directory, but unfortunately it seems we cannot really use bulk company names for testing, since many international companies use common anglo-american suffixes such as ltd. or inc. in their corporate names.

**Comment by davidheryanto** [comment]
Yes, agree with the Unicode approach. It will be applicable to company names in different countries.

The company names I gave are examples of companies in Singapore.

**Comment by petri** [comment]
We now have improved Unicode & non - Latin script support. So better test coverage would make sense too.

One option would be to use https://faker.readthedocs.io/en/master/ to generate fake test company names. Manual labour would still be needed to provide the expected base names that cleanco should be able to produce.
